tests/interfaces/test_jax.py

```diff
@@ -49,7 +56,7 @@ def test_raise_version_error(package, version, should_raise, monkeypatch):
         m.setattr(package, "__version__", version)
 
         if should_raise:
-            msg = "requires version 0.3.17 or higher for JAX and JAX lib"
+            msg = "requires version 0.3.17 or higher for JAX and 0.3.15 or higher JAX lib"

```

Comment:
```suggestion
            msg = "requires version 0.3.17 or higher for JAX and 0.3.15 or higher for jaxlib"
```
(if you go with the suggestion in the above file)